include/boost/uuid/string_generator.hpp

```diff
@@ -122,23 +133,27 @@ struct string_generator {
         static char const*const digits_end = digits_begin + 22;
```

Comment:
sizeof(digits_begin) if changed to ``static wchar_t const digits_begin[] = ...`` is 46, confirmed by a BOOST_STATIC_ASSERT, so in this case I think the hardcoded values is appropriate.  Considering the enhanced testing, any changes here will be caught.